core/src/main/kotlin/okuri/core/service/block/BlockEnvironmentService.kt

```
@@ -0,0 +1,445 @@
+                }
+            }
+        )
+    }
+
+    /**
+     * Handles overwriting the entire block environment in the event of a conflict.
+     * Would involve deleting all existing blocks and recreating them from the provided layout.
+     * To avoid situations where blocks were deleted in a conflicted version, and would no longer exist.
+     */
+    fun overwriteBlockEnvironment(request: OverwriteEnvironmentRequest): OverwriteEnvironmentResponse {
+        throw NotImplementedError()
```

detekt (1.23.8)

[warning] The NotImplementedDeclaration should only be used when a method stub is necessary. This defers the development of the functionality of this function. Hence, the `NotImplementedDeclaration` should only serve as a temporary declaration. Before releasing, this type of declaration should be removed.

(detekt.exceptions.NotImplementedDeclaration)